controller/array_action/array_mediator_ds8k.py

```diff
@@ -64,10 +64,12 @@ def scsi_id_to_volume_id(scsi_id):
     return scsi_id[-4:]
 
 
-def try_convert_first_arg(converter, args):
+def try_convert_firsts_arg(converter, args, num_of_args):
```

Comment:
`try_convert_first_args`, `number_of_args`/`args_amount`

Reply:
done

Reply:
`first_arg`**s** (plural), please avoid half words like `num`

Reply:
`arg` is a half-word as well

Reply:
but `args` can be found in official [PEP](https://www.python.org/dev/peps/pep-0448/#specification)s and docs, so I think it's legit

Reply:
In the official docs, they refer to `*args` / `**kwargs`, not to a "regular" parameter called `args`

Reply:
I don't think it matters if it's "regular" or not - it always means `arguments`, it's a common enough name in the language, so it should be easy for the reader to read

Reply:
I changed to args.. plz resolved it between yourself

Reply:
I think naming a parameter `args` without marking it with `*` is confusing.

Reply:
even with `*`, it just means `arguments`, like `argparse`. I don't see a reason to confuse it with the unrelated `*` operator.

Reply:
In Python `args` is kind of a saved keyword.
It's similar to have a function like:
`def foo(a, b, self): pass`
Usually, we don't use special names like that.

Reply:
`args` is not special as `self`.
`self` is important because otherwise the reader needs out-of-function context to know if the first arg is the class instance.
but the `*` (or lack of it) is visible in the same place, without extra context.
in addition, the general word "arguments" (or "args", for short) is more prevalent than the word "self", in coding.

Reply:
> the general word "arguments" (or "args", for short) is more prevalent than the word "self", in coding.
It's not in Python.
Not sure why you're against my suggestion. Do the extra five characters bother you?

Reply:
yes, and I think it's always more natural in python to use `args` instead of `arguments`

Reply:
as I see it, even with the fact that it's a `special name`, the `args` that are used here are the `args` of the special meaning. (unlike the `self` that clearly is not the context of the function)

Reply:
> the args that are used here are the args of the special meaning

I disagree. As long as there is no `*`, it's a regular parameter with confusing type and name.
Since both of you prefer to use `args`, I retreat.

Reply:
why confusing type? it's a tuple, just like with `*`

Reply:
How do you its type? You need to search for all the places we call this function first. With `*` you don't need to do that.

Reply:
with this naming convention, you don't need to search for all the places, just like `self`